src/main/java/it/pagopa/pn/deliverypush/dto/timeline/details/CategoryTypeTimelineElementDetailsInt.java

```diff
@@ -0,0 +1,14 @@
+package it.pagopa.pn.deliverypush.dto.timeline.details;
+
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@SuperBuilder( toBuilder = true )
+@ToString
+public class CategoryTypeTimelineElementDetailsInt {
```

Comment:
Una considerazione, questo viene esteso da tutti gli elementi di timeline?
Potrebbe aver senso non averlo specifico per il categoryType e avere qualcosa di generico tipo GenericTimelineElementDetails, dove ipoteticamente ci potrebbero finire tutti i campi presenti in tutti gli elementi, eventuali metodi comuni ecc.